README.md

```diff
@@ -232,11 +232,16 @@ These simulations are extremely useful for testing deployments before actually c
 
 #### Deploy Workflow
 
-0. Create the deployment script and configuration file, and test locally
-1. Open a PR containing the new deployment directory files
-2. Trigger the `deploy-market` workflow action through the GitHub UI
-3. Inspect the new `roots.json` which the workflow automatically commited to your PR
-4. Start using the new protocol deployment and/or create further migrations to modify it
+1. Create the deployment script and configuration file, and test locally
+2. Open a PR containing the new deployment directory files
+3. Trigger the `deploy-market` workflow action through the GitHub UI
+4. Inspect the new `roots.json` which the workflow automatically commited to your PR
+5. Start using the new protocol deployment and/or create further migrations to modify it
+
+##### Deploy Script Gotchas and Tips
+
+- If the deploy script is for a new market on a chain with an existing market, make sure to call 'setFactory(address,address)' in the initialization migration script.
+(TODO: Scenarios will fail because the factory will not be set during deployment, will need to figure out a better way)
```

Comment:
Can you add this: "Scenarios will fail prior to running the migration script " ...

Reply:
Sure no problem 👍